scripts/apps/desks/components/MarkBtn.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+/**
+ * @ngdoc React
+ * @module superdesk.apps.desks
+ * @name MarkBtn
+ * @param {Object} item the story to be marked
+ * @param {Object} desk desk for the item to be marked
+ * @param {Object} desk desksService
+ * @description Represents a button for given item to be marked for the given desk
+ */
+export class MarkBtn extends React.Component {
+    constructor(props) {
+        super(props);
+        this.item = props.item;
```

Comment:
These properties (`this.item`, `this.desk`, `this.desks`) they are not used anywhere. Am I missing something?